src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/LayerDescriptor.java

```diff
@@ -1,19 +1,19 @@
 package au.org.emii.geoserver.wms;
 
 public class LayerDescriptor {
-    public final String layer;
+    public final String groupLayerName;
```

Comment:
Yes as I had trouble understanding the variables in this file. Now they are all called what they are.